### PR TITLE
CUMULUS-2660: Update POST /executions endpoint

### PR DESCRIPTION
**Pull request by markdboyd**
**Summary:** Summary of changes

Addresses [CUMULUS-2660: Update POST /executions endpoint to post SNS messages and write to Elasticsearch](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2660)

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: packages/api/lib/writeRecords/write-execution.js [hunk 1]
```diff
@@ -131,6 +144,47 @@ const _writeExecutionRecord = ({
...
+      dynamoRecord,
+      postgresRecord,
+      knex,
+      executionModel,
+      esClient,
+      executionPgModel,
+      updatedAt,
+    }
+  );
+  const translatedExecution = await translatePostgresExecutionToApiExecution(
```
Comment (Jkovarik) [inline packages/api/lib/writeRecords/write-execution.js:180]: This is probably a dumb question we already discussed, but why are we recreating this translation given we already have the dynamo record, and the only functional addition is the createdAt/updatedAt values?  

Consider eliminating this translation, to eliminate needless postgres db load.     There's hidden debt/complexity in that change, so based on previous convo, I'm fine leaving it for a phase 3 reconsideration.

File: packages/api/tests/endpoints/test-executions.js [hunk 2]
```diff
@@ -1245,20 +1274,40 @@ test.serial('POST /executions creates the expected record', async (t) => {
...
   t.truthy(fetchedPgRecord.cumulus_id);
   t.is(fetchedPgRecord.async_operation_cumulus_id, t.context.asyncOperationCumulusId);
   t.is(fetchedPgRecord.collection_cumulus_id, t.context.collectionCumulusId);
   t.is(fetchedPgRecord.parent_cumulus_id, t.context.fakePGExecutions[1].cumulus_id);
 
-  const omitList = ['createdAt', 'updatedAt', 'created_at', 'updated_at', 'cumulus_id'];
   t.deepEqual(
-    omit(fetchedDynamoRecord, omitList),
-    omit(newExecution, omitList)
+    fetchedDynamoRecord,
```
Comment (Jkovarik) [inline packages/api/tests/endpoints/test-executions.js:1286]: 👍🏻

**Review by Jkovarik — APPROVED** [review]
This looks great!     Approving.

File: packages/api/lib/writeRecords/write-execution.js (hunk 1 above)
Comment (markdboyd) [reply packages/api/lib/writeRecords/write-execution.js:180]: Are we sure the timstamps are the only things that would have changed from the dynamo record? It's also possible that the `status` value would be unknown until after the record has been written, given the conditional logic. 

While I agree that the extra database lookups here aren't ideal, I feel like making partial updates to the dynamo record feels less safe and error-prone than doing the wholesale translation. 

Also, in phase 3, dynamo doesn't exist, so we can't really use that as the basis for what gets sent to the SNS topic.

**Pull request merged**